### ISSUE TITLE
VERR_INTNET_FLT_IF_NOT_FOUND E_FAIL issue on cf dev start on virtual box

### ISSUE THREAD

**Issue by nissshh** [vmware-archive/pcfdev#171]
Hello

I downloaded the zip file from the loacation "https://network.pivotal.io/products/pcfdev#/releases/3015" , unzipped the file and ran the exe (pcfdev-v0.22.0+PCF1.8.2-windows)  on windows 7 64 bit. After that i ran the command 'cf -dev start'.
It took a while for the VM to download and once done during the VM startup from Virtualbox got below error ----

```
Downloading VM...
Progress: |++++++++++++++++++==>| 100%
VM downloaded.
Allocating 4095 MB out of 8191 MB total system memory (4579 MB free).
Importing VM...
Starting VM...
←[31;1mFAILED←[0m
Error: failed to start VM: failed to execute 'C:\Program Files\Oracle\VirtualBox\VBoxManage startvm pcfdev-v0.389.0 --type headles
s': exit status 1: Waiting for VM "pcfdev-v0.389.0" to power on...
VBoxManage.exe: error: Failed to open/create the internal network 'HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #
2' (VERR_INTNET_FLT_IF_NOT_FOUND).
VBoxManage.exe: error: Failed to attach the network LUN (VERR_INTNET_FLT_IF_NOT_FOUND)
VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component ConsoleWrap, interface IConsole
.
```

I have no clue to what has been the issue. Does anyone face this issue before?

More , I started the app from VirtualBox UI and got below error  --

Failed to open/create the internal network 'HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #2' (VERR_INTNET_FLT_IF_NOT_FOUND).
Failed to attach the network LUN (VERR_INTNET_FLT_IF_NOT_FOUND).

```

Result Code: 
E_FAIL (0x80004005)
Component: 
ConsoleWrap
Interface: 
IConsole {872da645-4a9b-1727-bee2-5585105b9eed}

```
I am using VBox Version 5.1.8 r111374 (Qt5.5.1)

Regard
Nishant

**Comment by sclevine** [comment]
Hi @nissshh,

Sorry that you're experiencing this issue. We test PCF Dev extensively on 64-bit Windows 7 and have not seen this error. We're currently trying to reproduce it.

Could you run `cf dev destroy`, reboot your computer, and run `cf dev start` again?

We suspect that this is an issue with Virtualbox. If rebooting doesn't work, could you try installing Virtualbox 5.1.0 instead of 5.1.8?

**Comment by nissshh** [comment]
@sclevine Sure let me try that as you said,I'll update it once. Thanks

**Comment by nissshh** [comment]
@sclevine  It didnt work either , I destroyed the dev instance , it gave the same issue, I then installed the 5.1.0 version for VM (Version 5.1.0 r108711 to be specific) and tried to install the pcf dev instance again again the same error 

> C:\Windows\system32>cf dev start
Using existing image.
Allocating 4095 MB out of 8191 MB total system memory (4882 MB free).
Importing VM...
Starting VM...
←[31;1mFAILED←[0m
Error: failed to start VM: failed to execute 'C:\Program Files\Oracle\VirtualBox\VBoxManage startvm pcfdev-v0.389.0 --type headles
s': exit status 1: Waiting for VM "pcfdev-v0.389.0" to power on...
VBoxManage.exe: error: Failed to open/create the internal network 'HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #
2' (VERR_INTNET_FLT_IF_NOT_FOUND).
VBoxManage.exe: error: Failed to attach the network LUN (VERR_INTNET_FLT_IF_NOT_FOUND)
VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component ConsoleWrap, interface IConsole
.

**Comment by nissshh** [comment]
@sclevine While working on the solution  with refgerence to below posts 

https://caffinc.github.io/2015/11/fix-vbox-network/ and http://stackoverflow.com/questions/33725779/failed-to-open-create-the-internal-network-vagrant-on-windows10

I installed the Oravle VBox Version 5.1.0 r108711 (using Admin) but & with -msiparams NETWORKTYPE=NDIS5 

With above automatically the (VirtualBox Host-Only Network #2) was created when I did  `cf dev start`
 so enable /disable seems to be not a working option as specified in the posts URLs shared in this reply. It seems that installing with right MSI params and admin is a solution.


I've got things working with CLI and WebConsole working .

**Comment by nissshh** [comment]
@sclevine While working on the solution with refgerence to below posts

https://caffinc.github.io/2015/11/fix-vbox-network/ and http://stackoverflow.com/questions/33725779/failed-to-open-create-the-internal-network-vagrant-on-windows10

I installed the Oravle VBox Version 5.1.0 r108711 (using Admin) but & with -msiparams NETWORKTYPE=NDIS5

With above automatically the (VirtualBox Host-Only Network #2) was created when I did cf dev start
so enable /disable seems to be not a working option as specified in the posts URLs shared in this reply. It seems that installing with right MSI params and admin is a solution.

I've got things working with CLI and WebConsole working .

**Comment by aemengo** [comment]
@nissshh Thanks for posting the solution to your issue. It will help the team in the event that someone else runs into the same problem.

**Comment by shiva4weblogic** [comment]
Hi All
I tried the above solutions but didnt. Below are the details:-
C:\pcf\CloudFoundry>cf dev start
Using existing image.
Allocating 4096 MB out of 11984 MB total system memory (5325 MB free).
Importing VM...
Starting VM...
←[31;1mFAILED←[0m
Error: failed to start VM: failed to execute 'C:\Program Files\Oracle\VirtualBox\VBoxManage startvm pcfdev-v0.436.0 --ty
pe headless': exit status 1: Waiting for VM "pcfdev-v0.436.0" to power on...
VM "pcfdev-v0.436.0" has been successfully started.

Below is my software stack details:-
Oracle Virtualbox version: 5.0.12 r104815 (downloaded from virtualbox.org)

pcf version I downloaded is latest
C:\pcf\CloudFoundry>cf dev version
PCF Dev version 0.23.0 (CLI: 474b3ba, OVA: 0.436.0)

I spent nearly 3hrs ... any quick help is highly appreciated!!

Kindly, note that I tried the similar setup with Oracle Virtualbox 5.12.x.

Thanks,
Shiva
